forge-gui/res/cardsfolder/upcoming/oskar_rubbish_reclaimer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Oskar, Rubbish Reclaimer
+ManaCost:3 U B
+Types:Legendary Creature Human Wizard
+PT:3/3
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each different mana value among cards in your graveyard.
+SVar:X:Count$ValidGraveyard Card.YouOwn$DifferentCMC
+T:Mode$ Discarded | ValidPlayer$ You | ValidCard$ Card.nonLand | TriggerZones$ Battlefield | Execute$ TrigPlay | TriggerDescription$ Whenever you discard a nonland card, you may cast it from your graveyard.
+SVar:TrigPlay:DB$ Play | Valid$ TriggeredCard | ValidSA$ Spell | ValidZone$ Graveyard | Controller$ You | Optional$ True | Amount$ All

```

Comment:
@tool4ever @Northmoc shouldn't this be `Defined$ TriggeredCard` also should `ValidSA$ Spell.nonLand`?

because you might discard a MDC with a Land on the Backside, and you shouldn't be able to play the backside.

Reply:
I think `ValidSA$ Spell` will not match Lands already

Reply:
oky

Reply:
`Defined$ TriggeredCard` should work though, and then you wouldn't need `Valid` or `ValidZone` iirc